### ISSUE TITLE
Big difference in simulation results V10 <=> V11

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/PK-Sim#2224]
When loading this snapshot: huge difference in simulation results between V10 and V11 [Atazanavir-Model_1Sim.zip](https://github.com/Open-Systems-Pharmacology/PK-Sim/files/8631353/Atazanavir-Model_1Sim.zip)

Deactivating the Jacobian does not change the results. 
No particle dissolution formulations, no PgP

# V10
![grafik](https://user-images.githubusercontent.com/25061876/166927013-fab74c02-1973-484b-aad0-3878e9aec831.png)

# V11
![grafik](https://user-images.githubusercontent.com/25061876/166927044-a396fa2c-4dab-49e0-9e68-eb43b6fd3281.png)

**Comment by Yuri05** [comment]
The snapshot is a reduced version of https://github.com/Open-Systems-Pharmacology/Atazanavir-Model/blob/master/Atazanavir-Model.json
All simulations in that project produce deviations like that, also single admin.
[Open Systems Pharmacology Suite - 10-Installation Validation_05_05_22_14_28_25.pdf](https://github.com/Open-Systems-Pharmacology/PK-Sim/files/8631398/Open.Systems.Pharmacology.Suite.-.10-Installation.Validation_05_05_22_14_28_25.pdf)

**Comment by msevestre** [comment]
Hard to compare on different scal.e Can you put log as well?

**Comment by Yuri05** [comment]
V11 log
![grafik](https://user-images.githubusercontent.com/25061876/166939606-7a7117eb-2940-4996-af6f-2c5d6a8a4d10.png)

**Comment by Yuri05** [comment]
I think that's the only problem. When I set RefConc in V11 to the correct value - results are identical (at least visually)

**Comment by msevestre** [comment]
Investigating...

**Comment by msevestre** [comment]
project version "7.2.1".... cannot we upgrade at some point ?

**Comment by Yuri05** [comment]
most project snapshots were updated and saved with V10, except for:

* Projects with snapshot version 76 (PK-Sim 7 or 8)
  * Alfentanil
  * Atazanavir
  * Dapagliflozin
  * Fluvoxamine
  * Mefenamic-acid
  * Atazanavir-Raltegravir-DDI
  * Efavirenz-Alfentanil-DDI
  * Fluvoxamine-Alprazolam-DDI
  * Mefenamic_acid-Dapagliflozin-DDI

Projects with snapshot version 77 (PK-Sim 9)
  * Ethinylestradiol
  * Mexiletine
  * Moclobemide
  * Omeprazole
  * S-Mephenytoin
  * Tizanidine
 
Is it possible to fix or should we migrate the snapshots now?
After V11 release we will need to upgrade most of the snapshots anyway (e.g. in order to properly setup protein expression BBs)

**Comment by msevestre** [comment]
No no we should fix. But this is one of those project conversion stuff again. I can reproduce so that should be easy.

**Comment by msevestre** [comment]
but the value is not in the snapshot. Where was this 4.2 coming from?

**Comment by Yuri05** [comment]
it's the **default** value for CYP3A4 from PK-Sim DB

**Comment by msevestre** [comment]
Yeah but in v9 we introduced the fact that all those GLOBAL parameters should be saved in the json
With the v11 refactor, we have changed the way we are creating molecules and we are setting those global values after the fact. I could try to reintrdouce it but this feels kind of wrong.

**Comment by msevestre** [comment]
trying it out. Let see if this works
